logos/src/logos/capacity/capacity_planner.py

```
@@ -766,6 +925,11 @@ def _record_confirmed_action_state(
             self._lane_idle_since[key] = confirmed_at
             self._lane_loaded_at[key] = confirmed_at
             self._lane_was_cold_loaded[key] = (action.action == "load")
+            try:
+                from logos.monitoring import prometheus_metrics as prom
+                prom.CAPACITY_PLANNER_SWITCHES_TOTAL.inc()
+            except Exception:
+                pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)